src/sortable.js

```diff
@@ -154,8 +164,14 @@ angular.module('ui.sortable', [])
                 ui.item.sortable.dropindex = ui.item.index();
                 var droptarget = ui.item.parent();
                 ui.item.sortable.droptarget = droptarget;
-                var attr = droptarget.attr('ng-model') || droptarget.attr('data-ng-model');
-                ui.item.sortable.droptargetModel = droptarget.scope().$eval(attr);
+
+                var droptargetScope = null;
+                angular.forEach(ui.item.sortable._connectedSortables, function(x) {
+                  if (x.element[0] === droptarget[0]) {
+                    droptargetScope = x.scope;
```

Comment:
This was written at 7:30 am before I leave for work and obviously I was lazy enough to write a for loop...
I'm going to change it to use a for before merging and possibly refractor it to a getDroptargetScope function.